source/css/sass/patterns/molecules/teaser-list.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../variables/color";
+@import "../../mixins/decorations";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list__title {
+  @include block-spacing();
+  @include label-content-typography();
+  @include set-font-size-and-line-height(14px);
+  color: $color-text-normal;
+  font-weight: bold;
+  letter-spacing: 0.5px;
```

Comment:
The call chain for `@include label-content-typography()` visits [@mixin _label-typography](https://github.com/libero/pattern-library/blob/master/source/sass/mixins/_typography.scss#L85) which sets letter spacing, so `letter-spacing` doesn't need to be set here as well. (That said, the underlying variable `$font-letterspacing-label` is set to the wrong value at the moment, 1px rather than 0.5px, but that's a separate issue).